lib/mapper.js

```diff
@@ -239,7 +239,8 @@ function customParams(msg, eventName){
       fb_num_items: numberOfItems(msg),
       fb_content_id: contentIds(msg)
   };
-
+  // per FB docs, _appVersion is required in order to populate app version in event dashboard
+  if(msg.obj.context.app.version) customEvents._appVersion = msg.obj.context.app.version;
```

Comment:
Since it's recommended in the payload, I think it's fine to add app version to the test inputs.